scripts/masterCron.ts

```diff
@@ -558,12 +570,16 @@ async function strikeForecasting() {
 
 async function commitAndClose() {
   const vaultArtifact = await hre.artifacts.readArtifact("RibbonThetaVault");
-  const ierc20Artifact = await hre.artifacts.readArtifact(
-    "contracts/interfaces/IERC20Detailed.sol:IERC20Detailed"
-  );
 
   // 1. commitAndClose
   await runTX(vaultArtifact.abi, provider, signer, network, "commitAndClose");
+}
+
```

Comment:
decided to move token list update in rolltonextoption cron job. 

the reason is that in commitAndClose it will revert since I update token set without waiting for actual commitAndClose() to be mined (wait()). I could move it back and just wait() but then commitAndClose does not get called asynchronously so it is slower.